sklearn/ensemble/forest.py

```diff
@@ -1678,6 +1705,13 @@ class ExtraTreesRegressor(ForestRegressor):
         and add more estimators to the ensemble, otherwise, just fit a whole
         new forest. See :term:`the Glossary <warm_start>`.
 
+    ccp_alpha : positive float, optional (default=0.0)
```

Comment:
nit: isn't it technically non-negative float?

Reply:
I haven't heard anyone say that since grad school. 😅